src/main/java/core/basesyntax/RandomNumber.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RandomNumber {
+    public int getRandomNumber() {
+        return new Random().nextInt(100);
```

Comment:
random should be class field, 100 should be class constant, properly named, using CONSTANT_CASE